tests/test_pytest_operator.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from pathlib import Path
+
+tests_data_path = Path(__file__).parent / "data"
+tests_charms_path = tests_data_path / "charms"
+
+
+@pytest.mark.asyncio
+async def test_reactive_charm(operator_model, operator_tools):
```

Comment:
General question: Do we actually get anything out of these being `async`? Seems like a great approach if we can run multiple tests concurrently (does pytest do that automatically?), but unnecessary if each test is just run one at a time.

Reply:
I think the main reason we use async everywhere is b/c of libjuju. Unfortunately, async doesn't mean parallel tests in this regard but I do believe pytest has the ability to run each test in parallel if we chose to so we could have "best" of both worlds